.circleci/config.yml

```diff
@@ -103,6 +103,29 @@ jobs:
           environment:
             RUST_BACKTRACE: 1
           command: cargo test --verbose --frozen
+    
+  test_mockchain_all_features:
+    docker:
+      - image: rustlang/rust:nightly
```

Comment:
Are there unstable features so that you must use nightly?

Reply:
nope :) removing